File: URI/python/1180.py
n = int(raw_input())
lista = raw_input().split(' ')

menor = 10000
posicao = 0
for x in range(0, n):
  if(menor > int(lista[x])):
    menor = int(lista[x])
    posicao = x
print 'Menor valor:', menor
print 'Posicao:', posicao

File: URI/1180.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split("\n");
var tamanho = parseInt(linhas.shift());
var valores = linhas[0].split(" ").map(item=>parseInt(item)) ;

var min = valores[0];
var index = 0;
for (var i=0;i<valores.length;i++){
	if (valores[i]<min){
		min = valores[i];
		index = i;
	}
}

console.log("Menor valor: " + min);
console.log("Posicao: " + index)